news/2017-08-23-self-assessment-toolkit.html

```diff
@@ -0,0 +1,170 @@
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <div class="floe-content floe-news-item">
+                <h2> News </h2>
+                <h3>Self-Assessment Learning Toolkit</h3>
+                <p class="floe-date">August 23, 2017</p>
+                <p>
+                    We have been working on the design and implementation of an inclusive self-assessment toolkit that will engage learners themselves – supported by personalized learning analytics – as aspiring research scientists in the important subject of self-regulation and self-determination. The toolkit will engage a broad spectrum of learners in a self-directed, personalized learning process, by enabling learners to set flexible goals and customizable milestones towards achieving those goals based on their unique needs and preferences. 
+                </p>
+                <p>
+                    Recently, the design team has been considering alternate entry points into the toolkit, from highly specific to broad and exploratory. Part of this work has has been the exploration of new ways of engaging the learner through customizable “touchnotes”, which would afford the learner a quick and easy manual data collection process. A single data point is collected when a touchnote is selected. This can be combined with inputs from automated data collection sources (e.g. from a weather app or a personal health app, etc.), resulting in a fully personalizable data analysis and self-assessment.
+                </p>
+                <p>
+                    The following images show examples of the designs we are currently developing (select to enlarge):
+                </p>
+                
+                <figure>
+                    <a href="images/Self-reflection Figure 1.png"><img src="images/Self-reflection Figure 1 thumb.png" alt="view larger version of this image" aria-details="det1"></a>                    
+                    <figcaption> 
+                        <details id="det1">
+                            <summary>
+                            Figure 1. A learner can choose to be prompted for more detailed information when they select a touchnote in their personalized collection.
+                            </summary>
+                            <p>The image shows a collection of square 'touchnotes' buttons on the left with the heading 'Query: When do I feel most clear-headed and focused?'. The touchnotes in the collection include 'Focused', 'Good Idea', 'Meditation', 'Weather', 'Diet', and 'Sleep'. An arrow points from the 'Sleep' touchnote to a dialog box with prompts for more information including 'Time to bed', 'How many hours?', 'Woke up feeling: refreshed/tired', 'Disrupted? Y/N', and an option to add notes to a journal. There is a save button at the bottom of the dialog box.</p>
+                        </details>
+                    </figcaption>                 
+                </figure>       
```

Comment:
@klown, does the markup above look right?

Reply:
@jhung, LGTM.  It will be curious to see what ATs do with this.  *Crosses fingers*